### PR TITLE
Creates resource somtochi

### PR DESCRIPTION
**Pull request by somtochiama**
This PR creates a cluster-template-development instance

### CODE REVIEW

File: clusters/howard-moomboo-space/clusters/defau/somtochi.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
+apiVersion: gitops.weave.works/v1alpha1
+kind: GitopsCluster
+metadata:
+  annotations:
+    templates.weave.works/create-request: '{"head_branch":"wge-create-branch-whcgf","title":"Creates
+      resource somtochi","description":"This PR creates a cluster-template-development
+      instance","template_name":"cluster-template-development","parameter_values":{"CLUSTER_NAME":"somtochi","K3S_VERSION":"v1.25.3-k3s1","KUBERNETES_VERSION":"1.25.3","NAMESPACE":"defau"},"commit_message":"Creates
```
Comment (kingdonb) [inline clusters/howard-moomboo-space/clusters/defau/somtochi.yaml]: Looks like this was meant to be in the `default` namespace but it got `defau` by accident – it should be a dropdown 👍

File: clusters/howard-moomboo-space/clusters/defau/somtochi.yaml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    kustomize.toolkit.fluxcd.io/prune: disabled
+  labels:
+    cluster.x-k8s.io/cluster-name: somtochi
+    templates.weave.works/template-name: cluster-template-development
+    templates.weave.works/template-namespace: default
+  name: somtochi
+  namespace: defau
+spec:
+  controlPlaneEndpoint:
+    host: somtochi.turkey.local
```
Comment (kingdonb) [inline clusters/howard-moomboo-space/clusters/defau/somtochi.yaml]: I can set you up with an internal loadbalancer to go with this endpoint, or we can point a public domain at the public ip here so you can access the cluster directly from public internet

**Comment by kingdonb** [comment]
One additional change had to be made to this PR, as I forgot to fix it here:

https://github.com/kingdonb/bootstrap-repo/commit/bdf55a6b8989d67c4577752aec7f2c60fb6ce303

**Comment by somtochiama** [comment]
What the change just renaming the file

**Comment by kingdonb** [comment]
Yeah, without that it was not in the bootstrap path, so the rbac base did not get applied and access was blocked in WG dashboard... we fixed it though!

**Pull request merged**